packages/terra-clinical-item-view/docs/README.md

```diff
@@ -0,0 +1,52 @@
+# Terra Clinical Item View
+
+The Terra ClinicalItemView component allows displays to be organized into rows and column and themed, while providing means to add accessory elements.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-item-view`
+  - `yarn add terra-clinical-item-view`
+
+## React Usage
+
+The React component can be initialized like so:
+```
+import ClinicalItemView from 'ClinicalItemView;
```

Comment:
I think it should be this:
```
import ClinicalItemView from 'terra-clinical-item-view';
```

import ClinicalItemView from {name}, where {name} should match the [name attribute in the package.json](https://github.com/cerner/terra-core/pull/126/files#diff-d2ed4098563b24bea41c8808ecff8c2fR2)